visidata/statusbar.py

```diff
@@ -68,7 +68,7 @@ def leftStatus(sheet):
 @VisiData.api
 def drawLeftStatus(vd, scr, vs):
     'Draw left side of status bar.'
-    cattr = CursesAttr(colors.color_status)
+    cattr = ColorAttr(colors.color_status)
```

Comment:
Doesn't `colors.color_status` already return a ColorAttr?

Reply:
Nope! It has always returned a curses attribute.